migrations/sqls/20230921102640-delete-broken-charge-versions-down.sql

```diff
@@ -0,0 +1 @@
+/* Replace with your SQL commands */
```

Comment:
I blame the migration tool. But because it doesn't finish with a newline GitHub puts a 'shouty' red circle at the end to warn you.

```suggestion
/* Replace with your SQL commands */

```

Reply:
Looking at the changes we still don't have a new line at the end of this file. GitHub is still showing

![Screenshot 2023-09-21 at 12 55 01](https://github.com/DEFRA/water-abstraction-service/assets/1789650/3c5e928c-75b5-473a-90ff-ca9d547e8087)

If you commit my suggestion it will sort this. I also find that having `Insert Final Newline` ticked in my VSCode settings means I never commit a file without one.

![Screenshot 2023-09-21 at 12 56 26](https://github.com/DEFRA/water-abstraction-service/assets/1789650/985859f5-48e7-4f4f-aa2e-6d2674b45fb3)